xcode/Subconscious/Shared/Library/NaturalLanguage/TraceryDisambiguator.swift

```diff
@@ -12,57 +12,106 @@ struct TraceryContext: Hashable {
     var grammar: Grammar = [:]
 }
 
+enum DisambiguatorTag {
+    case journal
```

Comment:
I like the idea of note types supporting a taxonomy that is somehow more dynamic, open-ended, or data-defined. Perhaps we could have something like:

```
case noteKind(tags: [String])
```

and in the route we could use those string tags to look up keys in a Tracery Grammar, using the first one that is not nil, or defaulting to a generic grammar key.